cadence-style-guide.md

```diff
@@ -0,0 +1,65 @@
+# Cadence <- Style Guide
+## Table of contents
+1. [Identation](#identation)
+1. [Line length](#line-length)
+1. [Comments](#comments)
+1. [Whitespace between code lines](#whitespace-between-code-lines)
+1. [Declaring variables and constants](#whitespace-between-code-lines)
+1. [Bracket spaces](#bracket-spaces)
+1. [Panic messages](#panic-messages)
+1. [Naming and capitalization](#naming-and-capitalization)
+## Identation
```

Comment:
```suggestion
## Indentation
```

Reply:
well, I was actually pronouncing that badly also in Spanish (Indentación, not Identación xD)